src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/HealthCertificateValidationCoordinator.swift

```diff
@@ -45,6 +45,7 @@ final class HealthCertificateValidationCoordinator {
 
 	// MARK: Show Screens
 
+	// swiftlint:disable pattern_matching_keywords
```

Comment:
no opt out if you can comply with the rules

Reply:
Done